CONTRIBUTING.md

```diff
@@ -0,0 +1,284 @@
+# Contributing Guide
+
+## Welcome to Percona Server for MySQL!
+
+We're excited to have you join the Percona community and participate in keeping open-source open.
+
+### Why Your Contribution Matters?
+
+Main driving force of open-source project is community. We, as developers team, highly appreciate any contributions.
+
+It may be advisable to initially submit bug fixes to the upstream project that will eventually become part of the next
+release of Percona. Alternatively, Percona provides expedited response times, transparent development processes, and
+streamlined contribution acceptance within a public repository.  
+You can contribute in one of the following ways:
+
+* Submit a bug report or a feature
+   request in [ our public Jira ](https://jira.percona.com/projects/PS/issues).
+* Submit a pull request (PR) with the code patch.
+* Contribute to [the documentation](https://github.com/percona/psmysql-docs/blob/innovation-release/contributing.md)
+* Reach us on our [ Forums ](https://forums.percona.com/c/mysql-mariadb)
+
+This document describes the workflow for submitting pull requests.
+By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/). 
+
+### Steps of the workflow
+
+Steps of the workflow for submitting pull request are:
+
+1. Creating Jira issue
+2. Forking of Percona repository
+3. Creating a branch for your changes
+4. Building code before changes
+5. Opening the code in an IDE, making changes and debugging
+6. Testing changes
+7. Preparing the pull request
+8. Peer review and required fixes
+9. Merging pull request
```

Comment:
Will the contributor merge the pull request?

Reply:
Usually, yes, if the contributor has rights. If no, we do.  It is explained in the corresponding paragraph.